src/Components/DarkMode/DarkMode.js

```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from '../../contexts/ThemeContext';
+
+function DarkMode() {
+  const { toggleDarkMode } = useContext(ThemeContext)
+  return (
+    <>
+      <button onClick={() => toggleDarkMode()}>Dark Mode</button>
```

Comment:
This solution is the team work I'm proud to be a part of, simple, minimal.  Good job team.